charts/seed-bootstrap/charts/loki/templates/statefulset.yaml

```diff
@@ -16,15 +16,66 @@ spec:
     metadata:
       labels:
         networking.gardener.cloud/from-prometheus: allowed
+{{- if .Values.rbacSidecarEnabled }}
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-shoot-apiserver: allowed
+{{- end}}
 {{ toYaml .Values.labels | indent 8 }}
       annotations:
         checksum/configmap-loki-config: {{ include (print $.Template.BasePath "/loki-configmap.yaml") . | sha256sum }}
+{{- if .Values.rbacSidecarEnabled }}
+        checksum/kube-rbac-proxy-kubeconfig: {{ .Values.kubeRBACProxyKubeconfigCheckSum }}
+        checksum/configmap-telegraf-config: {{ include (print $.Template.BasePath "/telegraf-config.yaml") . | sha256sum }}
+{{- end}}
     spec:
       serviceAccountName: loki
       securityContext:
-        {{- toYaml .Values.securityContext | nindent 8 }}
+        fsGroup: 10001
       priorityClassName: {{ .Values.priorityClass.name }}
       containers:
+{{- if .Values.rbacSidecarEnabled }}
+        - name: kube-rbac-proxy
```

Comment:
Can you assign resources to this container. Also the HVPA should be modified to not monitor this container. See https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/charts/loki/templates/hvpa.yaml#L77

Reply:
I have assigned a resource limits and requests base on the following graph.
![Screenshot 2021-06-16 at 15 03 15](https://user-images.githubusercontent.com/38783428/122584036-8ca61880-d062-11eb-8581-90d23d05cda5.png)
![Screenshot 2021-06-16 at 15 09 32](https://user-images.githubusercontent.com/38783428/122584057-90d23600-d062-11eb-8f4b-91755c8e8eca.png)

Here the Shoot was scalling from 2 to 20 and then to 50 nodes for more than 30 minutes